iidm/iidm-api/src/main/java/com/powsybl/iidm/network/RatioTapChangerHolder.java

```diff
@@ -21,6 +21,34 @@ public interface RatioTapChangerHolder {
      */
     RatioTapChangerAdder newRatioTapChanger();
 
+    /**
+     * Get a builder to create and associate a ratio tap changer to the
+     * transformer, initialized with the values of an existing ratio tap changer.
+     */
+    default RatioTapChangerAdder newRatioTapChanger(RatioTapChanger ratioTapChanger) {
+        RatioTapChangerAdder adder = this.newRatioTapChanger()
+                .setRegulationTerminal(ratioTapChanger.getRegulationTerminal())
+                .setRegulationMode(ratioTapChanger.getRegulationMode())
+                .setRegulationValue(ratioTapChanger.getRegulationValue())
+                .setLoadTapChangingCapabilities(ratioTapChanger.hasLoadTapChangingCapabilities())
+                .setTargetV(ratioTapChanger.getTargetV())
+                .setLowTapPosition(ratioTapChanger.getLowTapPosition())
+                .setTapPosition(ratioTapChanger.getTapPosition())
+                .setRegulating(ratioTapChanger.isRegulating())
+                .setTargetDeadband(ratioTapChanger.getTargetDeadband());
+        for (int tapPosition = ratioTapChanger.getLowTapPosition(); tapPosition <= ratioTapChanger.getHighTapPosition(); tapPosition++) {
+            RatioTapChangerStep step = ratioTapChanger.getStep(tapPosition);
+            adder.beginStep()
```

Comment:
wouldn't it be interesting to add a `A beginStep(D step)` in `TapChangerAdder` interface? It's probably not very useful, but in that way each level would have its copy method.

Reply:
I would prefer to stick to the perimeter you defined previously.

If the need for copy-adder on steps solidifies, they could be added in a separate PR.

Reply:
After looking back into it, the issue I have with it is that `beginStep` carries a state that would be unrelated to existing step: the order in which steps are created currently encodes the tap position. Being able to initialize `beginStep` by copying a step with a different tap position may be confusing for users.

A helper class could be used to mutualize the code, but it is unclear to me whether this would be better than the current situation.